README.md

```diff
@@ -3,5 +3,6 @@ This repo contains code for blog posts from [OpenCV.AI](https://opencv.ai)
 
 | Blog Post | |
 | ------------- |:-------------|
+|[Modelplace Cloud API](https://modelplace.ai/blog )| [Code](https://github.com/opencv-ai/opencv-blog/tree/main) |
```

Comment:
Wrong link for the code

`https://github.com/opencv-ai/opencv-blog/tree/main/Modelplace-Cloud-API-Python` is the right one

Please, use `https://modelplace.ai/blog/cloud-api` for `Modelplace Cloud API`